### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by hakanaku1234** [evertramos/nginx-proxy-automation#326] [Q&A]
hello, i have started this project in background successfully

```
Creating letsencrypt-auto ... done
Creating docker-gen-auto  ... done
Creating proxy-web-auto   ... done
 ✔ success Function 'docker_compose_start'
 ✔ success Your proxy was started successfully!
```

but when i access my sub-domain (ex: test1.example.com, test2.example.com, etc...)
the browser (chrome) still response `DNS_PROBE_FINISHED_NXDOMAIN` 
looks like it didn't reach the nginx proxy.

i wonder how can i fix this problem, thanks!

**Comment by evertramos** [comment]
Hello @hakanaku1234!

This proxy is a service which you will use along with your sites, so, you must add your new service/site (test1.example.com) in your new docker ccontainer using the environment variables:

```bash
  VIRTUAL_HOST=your.domain.com
  LETSENCRYPT_HOST=your.domain.com
  LETSENCRYPT_EMAIL=your.email@your.domain.com
```

So, if your domain is set to your server correctly you will be able to access your domain in your browser using the following:

```bash
docker run -dit -e VIRTUAL_HOST=your.domain.com -e LETSENCRYPT_HOST=your.domain.com -e LETSENCRYPT_EMAIL=you@your.domain.com --network=proxy --name test-web httpd:alpine
```

Please follow through the docs below and see if there is any remaining questions:

https://github.com/evertramos/nginx-proxy-automation/tree/master/docs

Thanks!
